src/hdapsd.c

```diff
@@ -310,7 +316,7 @@ static int read_position_from_inputdev (int *x, int *y, int *z, double *utime)
 				continue;
 		}
 		if (!*utime) /* first event's time is closest to reality */
-			*utime = ev.time.tv_sec + ev.time.tv_usec/1000000.0;
+			*utime = ev.input_event_sec + ev.input_event_usec/1000000.0;
```

Comment:
on 64 bits systems this is a no-op as `input_event_*` points at `time.tv_*` (see https://github.com/torvalds/linux/commit/152194fe9c3f03232b9c0d0264793a7fa4af82f8#diff-0028ed2bcc0d6c2118b5b33c6f3bf7d398e3df872553183ac2c0abebba562f01 which introduced that).

whether this *actually* works on 32 bit systems? I have no idea. it compiles and throws no warnings.

my private ppc32 system is acting up a bit, so I can't really test it rn.